### PR TITLE
Hporutiu readme updates to node.js stickies CSV app

### PR DESCRIPTION
**Pull request by horeaporutiu**
* add demo video
* add sample CSV data
* update folder structure with csv data file
* use README template
* added section to CONTRIBUTING.md to ensure all new app-examples which are submitted have the required sections

### CODE REVIEW

**Review by mettin — APPROVED** [review]

**Comment by horeaporutiu** [comment]
also added a new section in the contributing guide @mettin could you please check that when you get a chance?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -99,3 +99,25 @@ Congratulations! The Miro team is happy to have you contribute.
...
+7. [Run the app section](https://github.com/miroapp/app-examples/tree/main/examples/drag-and-drop#%EF%B8%8F-run-the-app-locally-). This section should go over how to run the app locally. Please be as descriptive as possible to ensure the developer can install
+   and configure your app easily, even if it is their first time using these tools and technologies.
+8. [Folder Structure section](https://github.com/miroapp/app-examples/tree/main/examples/drag-and-drop#%EF%B8%8F-folder-structure-). This section outlines what each file is for in the app so a developer can quickly
+   understand the file layout of your app.
+9. [Contributing section](https://github.com/miroapp/app-examples/tree/main/examples/drag-and-drop#-contributing-). Here you
+   can just link to [Miro's contributing guide](https://github.com/miroapp/app-examples/blob/main/CONTRIBUTING.md).
+10. [License section](https://github.com/miroapp/app-examples/tree/main/examples/drag-and-drop#-license-). This should
+    indicate what license you want to use.
```
Comment (mettin) [inline CONTRIBUTING.md:123]: Can we create a README.template.md too? 🔥

Comment (horeaporutiu) [reply CONTRIBUTING.md:123]: Yes, let me do that now :)

Comment (horeaporutiu) [reply CONTRIBUTING.md:126]: Ok it's added now @mettin! What do you think?

File: README.template.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+https://github.com/horeaporutiu/app-examples-template/assets/10428517/b23d9c4c-e785-43f9-a72e-fa5d82c7b019
+
+# 🗂️ Folder structure <a name="folder"></a>
+
+<b>Explain your folder structure so a developer can quickly understand the layout of your code.</b>
+
+```
+.
+├── src
+│  └── styles
```
Comment (mettin) [inline README.template.md:80]: You can delete these lines since its too specific

Comment (mettin) [reply README.template.md:80]: 80-84

File: README.template.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+> <b>Add steps to run the app locally. Try to make them as similar as possible to other example apps for consistency.</b>
+
+1. Run `npm install` to install dependencies.
+2. Run `npm start` to start developing. \
+   Your URL should be similar to this example:
+   ```
+   http://localhost:3000
+   ```
+3. Open the [app manifest editor](https://developers.miro.com/docs/manually-create-an-app#step-2-configure-your-app-in-miro) by clicking **Edit in Manifest**. \
+   In the app manifest editor, configure the app as follows:
```
Comment (mettin) [inline README.template.md:52]: can't we have a copy/paste example?

Comment (horeaporutiu) [reply README.template.md:52]: I added one, lmk if it is what you were thinking? @mettin

File: README.template.md (hunk 2 above)
Comment (horeaporutiu) [reply README.template.md]: Good idea, just removed. Thank you!

**Review by kirillsud — APPROVED** [review]

**Pull request merged**